notebooks/Tutorial_2_train_your_first_TTS_model.ipynb

```diff
@@ -0,0 +1,438 @@
+   "execution_count": null,
+   "id": "0f609945-4fe0-4d0d-b95e-11d7bfb63ebe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from trainer import Trainer, TrainerArgs\n",
+    "trainer = Trainer(\n",
+    "    TrainerArgs(), config, output_path, model=model, train_samples=train_samples, eval_samples=eval_samples\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b320831-dd83-429b-bb6a-473f9d49d321",
+   "metadata": {},
+   "source": [
+    "### AND... 3,2,1... START TRAINING 🚀🚀🚀"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4c07f99-3d1d-4bea-801e-9f33bbff0e9f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainer.fit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cff0c40-2734-40a6-a905-e945a9fb3e98",
+   "metadata": {},
+   "source": [
+    "#### 🚀 Run the Tensorboard. 🚀\n",
+    "On the notebook and Tensorboard, you can monitor the progress of your model. Also Tensorboard provides certain figures and sample outputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a85cd3b-1646-40ad-a6c2-49323e08eeec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install tensorboard\n",
+    "!tensorboard --logdir=tts_train_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f6dc959",
+   "metadata": {},
+   "source": [
+    "## ✅ Test the model\n",
+    "\n",
+    "We made it! 🙌\n",
+    "\n",
+    "Let's kick off the testing run, which displays performance metrics.\n",
+    "\n",
+    "We're committing the cardinal sin of ML 😈 (aka - testing on our training data) so you don't want to deploy this model into production. In this notebook we're focusing on the workflow itself, so it's forgivable 😇\n",
+    "\n",
+    "You can see from the test output that our tiny model has overfit to the data, and basically memorized this one sentence.\n",
+    "\n",
+    "When you start training your own models, make sure your testing data doesn't include your training data 😅"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99fada7a-592f-4a09-9369-e6f3d82de3a0",
+   "metadata": {},
+   "source": [
+    "Let's get the latest saved checkpoint. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6dd47ed5-da8e-4bf9-b524-d686630d6961",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob, os\n",
+    "output_path = \"tts_train_dir\"\n",
+    "ckpts = sorted([f for f in glob.glob(output_path+\"/*/*.pth\")])\n",
+    "configs = sorted([f for f in glob.glob(output_path+\"/*/*.json\")])\n",
+    "os.environ['test_ckpt'] = ckpts[-1]\n",
+    "os.environ['test_config'] = configs[-1]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd42bc7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " !tts --text \"Text for TTS\" \\\n",
+    "      --model_path $test_ckpt \\\n",
+    "      --config_path $test_config \\\n",
```

Comment:
Jupyter lets you access Python variables in inline shell calls like this, so you don't have to set them in `os.environ` above, just create normal Python variables `test_ckpt` and `test_config`.